### PR TITLE
update book link and remove obsolete directory

### PR DESCRIPTION
**Pull request by tersec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Nimbus beacon chain is a research implementation of the beacon chain component o
 
 You can find the information you need to run a beacon node and operate as a validator in [The Book](https://status-im.github.io/nimbus-eth2/).
 
-See [this page](https://status-im.github.io/nimbus-eth2/medalla.html) for how to become a validator on Medalla.
+See [this page](https://status-im.github.io/nimbus-eth2/) for how to become a validator on Medalla.
```
Comment (arnetheduck) [inline README.md:52]: remove this line completely?

Comment (tersec) [reply README.md]: https://github.com/status-im/nimbus-eth2/pull/2097/commits/145062f138a61a0b5e70d90d3531c0daffe99dc7

**Comment by arnetheduck** [comment]
btw isn't this docker used somewhere? @stefantalpalaru ?

**Comment by tersec** [comment]
I don't know; I asked in Discord. The purpose of this PR was to get that kind of feedback. It seems unlikely due to all the references to `witti` and `medalla`. I don't see how it could possibly have been working. Maybe people have their own local patches on top of this.

**Comment by tersec** [comment]
None of the AWS hosts from `medalla_validators` even resolves anymore:
```
$ for h in master-01.aws-eu-central-1a.nimbus.test.statusim.net node-01.aws-eu-central-1a.nimbus.test.statusim.net node-02.aws-eu-central-1a.nimbus.test.statusim.net node-03.aws-eu-central-1a.nimbus.test.statusim.net node-04.aws-eu-central-1a.nimbus.test.statusim.net node-05.aws-eu-central-1a.nimbus.test.statusim.net; do ssh $h; done
ssh: Could not resolve hostname master-01.aws-eu-central-1a.nimbus.test.statusim.net: Name or service not known
ssh: Could not resolve hostname node-01.aws-eu-central-1a.nimbus.test.statusim.net: Name or service not known
ssh: Could not resolve hostname node-02.aws-eu-central-1a.nimbus.test.statusim.net: Name or service not known
ssh: Could not resolve hostname node-03.aws-eu-central-1a.nimbus.test.statusim.net: Name or service not known
ssh: Could not resolve hostname node-04.aws-eu-central-1a.nimbus.test.statusim.net: Name or service not known
ssh: Could not resolve hostname node-05.aws-eu-central-1a.nimbus.test.statusim.net: Name or service not known
```

`entry_point.sh` only mentions `pyrmont` instead of `medalla` because I updated it recently.

Et cetera.

**Comment by stefantalpalaru** [comment]
> btw isn't this docker used somewhere? @stefantalpalaru ?

I don't think so.

**Comment by tersec** [comment]
@jakubgs do you use any of these?

**Pull request merged**